index.html

```diff
@@ -63,5 +63,169 @@ <h2>I’m a software developer</h2>
             </div>
         </div>
     </section>
+    <section class="page-1">
+        <div class="header">
+            <h1>My Recent Works</h1>
+        </div>
+        <br>
+        <div class="line">
+            <hr>
+        </div>
+        <div class="grid">
+        <div class="grid1"></div>
+        <div class="grid2">
+            <br>
+            <h1>Multi-Post Stories <br> Gain+Glory</h1>
+            <br>
+            <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>html</li>
+            </ul>
+            <br>
+            <button>See Project</button>
+            <br>
+        </div>
+        </div>
+        <div class="grid">
+        <div class="grid1"></div>
```

Comment:
- [ ] Kindly make to use h1 only once on the pages in order to respect the HTML/CSS best practices :recycle: